cli/mod.mjs

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
```

Comment:
I don't see the value of one "uber" mod CLI. I also personally don't enjoy running terminal-based commands directly ;-) . Granted, not all are.